.github/workflows/main.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         make \
         software-properties-common \
         git build-essential gcc pkg-config cmake \
-        libmbedtls-dev
```

Comment:
OpenSSL is only supported in open62541 1.3.  We should keep mbedtls for earlier versions.